include/seqan3/argument_parser/validators.hpp

```diff
@@ -373,23 +373,27 @@ class file_validator_base
             throw validation_error{detail::to_string("The given filename ", path.string(), " has no extension. Expected"
                                                      " one of the following valid extensions:", extensions, "!")};
 
-        // Drop the dot.
-        std::string drop_less_ext = path.extension().string().substr(1);
+        std::string const path_as_string{path.filename().string()};
+        std::string_view file_path{path_as_string};
```

Comment:
Not sure about all of this string_views, do we really need them?

I think it would be fine to use strings here.